src/centos/scripts/start-marklogic.sh

```diff
@@ -28,13 +28,23 @@ for var in OVERWRITE_ML_CONF INSTALL_CONVERTERS MARKLOGIC_DEV_BUILD MARKLOGIC_IN
     declare $var="$(echo "${!var}" | sed -e 's/[[:blank:]]//g' | awk '{print tolower($0)}')"
 done
 
-# define log and err functions
-log() {
-    echo "$(basename "${0}"): ${*}"
+#log utility
+info() {
+    log "Info" "$@"
```

Comment:
If there is no debug level (or some other second non error log level) what is the point of the info level? If everything that used to be log is now info why wouldn't we just leave it as log?

Reply:
Currently, we don't have debug level logs but the log messages will be reviewed to identify the log levels as part of this story. Also, scripts should follow the standard logging levels info, error, debug, etc.